images/docker-compose-motorpigcs2.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  ioc:
+    image: ghcr.io/cnpem/motorpigcs2-epics-ioc:$TAG
```

Comment:
Name this `ghcr.io/cnpem/motor-pigcs2-epics-ioc:$TAG`. This will make motor IOCs follow a similar convention that we use with areaDetector IOCs, whose names are `ad-<module>-epics-ioc`.